### PR TITLE
highlighted doxygen questions for R. J. Purser in sorc/grid_tools.fd/regional_esg_grid.fd code

### PR DESCRIPTION
**Pull request by edwardhartnett**
I have asked the author to contribute some info. This PR is to highlight the missing fields we need filled in.

Part of #191 
Part of #343 

@JimPurser-NOAA this is for you. ;-)

### CODE REVIEW

File: sorc/grid_tools.fd/regional_esg_grid.fd/pesg.f90 [hunk 1]
```diff
@@ -130,10 +130,10 @@ end subroutine xstoxc1
 
 !> Inverse of xttoxs.
 !!
-!! @param[in] k ???
-!! @param[in] xs ???
-!! @param[out] xt ???
-!! @param[out] ff ???
+!! @param[in] k ???
```
Comment (JimPurser-NOAA) [inline sorc/grid_tools.fd/regional_esg_grid.fd/pesg.f90:133]: Gauss curvature parameter of mapping

**Comment by edwardhartnett** [comment]
I've emailed this code file to Jim and he is editing it, and will mail it back to me, and I will update this PR.

**Comment by edwardhartnett** [comment]
Jim emailed me back the edited file, and I have added his very helpful edits to the PR. I've also emailed him more files, so hopefully we'll be adding more documentation to the rest of regional_esg_grid.fd code. ;-)

**Comment by edwardhartnett** [comment]
@kgerheiser let's get this reviewed and merged, and I will put up a new PR with the next batch of edits from R. Purser when I get them.

**Review by GeorgeGayno-NOAA — APPROVED** [review]
Looks good.  Builds without doxygen warnings.

**Pull request merged**